Show base branch in PR title

JakobDev opened the pull request:
f-e-d-c is currently used by [com.riverbankcomputing.PyQt.BaseApp](https://github.com/flathub/com.riverbankcomputing.PyQt.BaseApp) on 3 different branches. Having the branch (e.g. 5.15-22.08 or 6.3) would make it possible to see which branch is targeted directly in the Notification or the PR list.

Note: This is currently not tested

wjt commented on src/main.py:
```diff
@@ -247,6 +248,11 @@ def open_pr(
     head = "{}:{}".format(repo.owner.login, change.branch)
     pr_message = ((change.body or "") + "\n\n" + DISCLAIMER).strip()
+    
+    if branch_title:
```
or something like:

```suggestion
    if base not in ('master', 'beta'):
```

JakobDev replied:
I personal prefer a argument, so the User can decide if he want it or not. There may be someone who uses f-e-d-c on the master and the beta branch and want to know, if a PR goes against the master or the beta branch.

gasinvein commented on src/main.py:
```diff
@@ -210,6 +210,7 @@ def open_pr(
     change: CommittedChanges,
     manifest_checker: manifest.ManifestChecker = None,
     fork: t.Optional[bool] = None,
+    branch_title: t.Optional[bool] = None,
```
```suggestion
    branch_title: bool = False,
```

bbhtt committed:
main: Include base branch in PR title

If it is different than the origin repo's default branch

bbhtt committed:
main: Use keyword args for create_pull

The order of the arguments changed in PyGitHub's current version
so newer documentation is confusing to look at.

This was the order used in version closest to docker image (1.55)

https://pygithub.readthedocs.io/en/v1.58.0/github_objects/Repository.html?highlight=create_pull#github.Repository.Repository.create_pull
